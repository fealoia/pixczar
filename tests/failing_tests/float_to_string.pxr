Void main() {
	Float x = 3.5;
	x = "hello";
	print(x);
}
