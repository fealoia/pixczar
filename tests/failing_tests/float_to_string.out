Fatal error: exception Failure("illegal assignment Float = String in x = "hello"")
